rts/lra/lra-service-base/src/main/java/io/narayana/lra/LRAHttpClient.java

```diff
@@ -19,7 +19,7 @@
 import java.util.concurrent.TimeoutException;
 
 public class LRAHttpClient {
-    public static final long PARTICIPANT_TIMEOUT = 1; // number of seconds to wait for requests
+    public static final long PARTICIPANT_TIMEOUT = 10; // number of seconds to wait for requests
```

Comment:
Well 1 second should be more than enough but the new test on the existing CI infrastructure intermittently fails because the 1 second time limit is breached. I have not studied your test in detail to find out why the callbacks sometimes take longer than 1 second (or whether the test is making certain assumptions) but I made it 10 s for expediency. Note that the shorter timelimit should work if the tests run sufficient recovery passes (but I did not spend any time investigating).

Reply:
I have changed it to 2 seconds. When we take the latest MP LRA snapshot we can go back to 1 second since the latest TCK runs a recovery pass which would pick it up. The  RC1 TCK doesn't retry you see (also see my [earlier comment](https://github.com/jbosstm/narayana/pull/1559#issuecomment-581549705)).